### ISSUE TITLE
Quectel EC25: parameter definition error IMSI and IMEI

### ISSUE THREAD

**Issue by Harwesta** [koshev-msk/luci-app-modeminfo#45]
Hi.

I found a parameter parsing error for Quectel EC25.
When IMEI is requested, 'OK' is displayed, the real IMEI is shown in the IMSI field (see screenshot).
Real IMSI not displayed.

Installed packets version:
luci-app-modeminfo v0.3.0-1
modeminfo v0.3.0-5
modeminfo-qmi v0.3.0-5
modeminfo-serial-quectel v0.3.0-5

![EC25_bug](https://user-images.githubusercontent.com/65107625/183357157-ea789b4f-3fdb-44ff-aef3-428178c95164.jpg)

**Comment by koshev-msk** [comment]
Please send to me command output atinout /usr/share/modeminfo/scripts/QUECTEL.at /dev/tty*MODEMPORT* -

**Comment by Harwesta** [comment]
Done!

```
ATE0


OK



OK



+COPS: 0,2,"25099",7



+CREG: 0,1



+CGREG: 0,1



+CGREG: 0,1



+CEREG: 0,1



+CSQ: 30,99



OK



+QENG: "servingcell","NOCONN","LTE","FDD",250,99,A1A9566,113,550,1,3,3,61EA,-84,-11,-56,2,-



OK



ERROR



+QTEMP: 42,38,38

OK

```

**Comment by koshev-msk** [comment]
Strange output. I don't see vendor, model, iccid, imsi and firmware version, but on sceen viewed it.
Okay, replace line 15 in script /usr/share/modeminfo/scripts/QUECTEL to

```
*EC25*|*EP06*)
```

**Comment by Harwesta** [comment]
Maybe this parameters (vendor, model, iccid, imsi etc) request and parsing another one script?

**Comment by koshev-msk** [comment]
Fixed in 2f1fdeb145cb22ccc8f725816affd5f7faf40e51
